### ISSUE TITLE
Include fping6 binary for IPv6 support

### ISSUE THREAD

**Issue by electricOzone** [linuxserver/docker-smokeping#84]
## Expected Behavior
fping6 binary should be included in image for pinging IPv6 targets.

## Current Behavior
fping6 is not present under /usr/bin or /usr/sbin

## Steps to Reproduce
ls -la in /usr/bin and /usr/sbin

## Environment
**OS:**      Ubuntu 18.04.4 LTS
**CPU architecture:** x86_64
**How docker service was installed:**      sudo apt install docker docker-compose

## Command used to create docker container (run/create/compose/screenshot)
```
  smokeping:
    image: linuxserver/smokeping
    container_name: smokeping
    volumes:
      - /opt/docker/smokeping/config:/config
      - /opt/docker/smokeping/data:/data
    environment:
      - PUID=123
      - PGID=123
    restart: always
```

## Docker logs

```
-------------------------------------
          _         ()
         | |  ___   _    __
         | | / __| | |  /  \ 
         | | \__ \ | | | () |
         |_| |___/ |_|  \__/


Brought to you by linuxserver.io
We gratefully accept donations at:
https://www.linuxserver.io/donate/
-------------------------------------
GID/UID
-------------------------------------

User uid:    999
User gid:    999
-------------------------------------

[cont-init.d] 10-adduser: exited 0.
[cont-init.d] 30-config: executing... 
cp: cannot stat '/config/smtp.conf': No such file or directory
[cont-init.d] 30-config: exited 0.
[cont-init.d] 99-custom-files: executing... 
[custom-init] no custom files found exiting...
[cont-init.d] 99-custom-files: exited 0.
[cont-init.d] done.
[services.d] starting services
AH00558: httpd: Could not reliably determine the server's fully qualified domain name, using 172.18.0.3. Set the 'ServerName' directive globally to suppress this message
[services.d] done.
Smokeping version 2.007003 successfully launched.
Entering multiprocess mode.
### parsing dig output...OK
Child process 284 started for probe FPing.
No targets defined for probe DNS, skipping.
All probe processes started successfully.
FPing: probing 32 targets with step 300 s and offset 170 s.
```

**Comment by thelamer** [comment]
Upstream Alpine specifically pulled the build arg in this commit for ipv6, compiling locally I am not able to get an fping6 binary. Is this just something that is built into the main bin now? 
https://git.alpinelinux.org/aports/commit/main/fping?id=6022814d38d6ad1d3cc60bce55e5f8cb444518e7

**Comment by electricOzone** [comment]
I tried just using the regular fping binary (I didn't specify a probe), with an IPv6 host address and this doesn't seem to work. For example:
```
+ IPv6
menu = IPv6
title = IPv6

++ GoogleDNSIPv6
menu = GoogleDNSIPv6
title = GoogleDNSIPv6
host = 2001:4860:4860::8888
```

**Comment by electricOzone** [comment]
Output from container shell:
```
root@d8eb11d2a6bf:/# fping 2001:4860:4860::8888
2001:4860:4860::8888: error while sending ping: Address not available
```

**Comment by electricOzone** [comment]
Ah, it looks like in my case, I'm having trouble getting IPv6 running with docker-compose. I'm not sure if the fping binary is capable of IPv6, but since fping --help shows the commands for IPv6, I imagine it is. I'll go ahead and close this issue for now.

**Comment by crashGrumpf** [comment]
fping6 is merged with fping on alpine 3.11 enable ipv6 on docker bridge then 

you should set Probe: (not working without protocol = 6)
`+FPing6

binary = /usr/sbin/fping
protocol = 6
`
Target
`++ GoogleDNS1
probe = FPing6
menu = Google DNS 1
title = Google DNS 2001:4860:4860::8888
host = 2001:4860:4860::8888
`
